portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    >
+    <!-- Bootstrap JS, Popper.js, and jQuery -->
+    <script
+        src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+        crossorigin="anonymous">
+    </script>
+    <script
+        src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous">
+    </script>
+    <script
+        src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+        integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+        crossorigin="anonymous">
+    </script>
+    <!-- FontAwesome Icons -->
+    <script
+        src="https://kit.fontawesome.com/7cf05ff215.js"
+        crossorigin="anonymous">
+    </script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="page-container">
+      <div id="navbar">
+        <nav class="nav flex-column">
+          <a class="nav-link" href="index.html">Home</a>
+          <a class="nav-link" href="about.html">About</a>
+          <a class="nav-link" href="blog.html">Blog</a>
+          <a class="nav-link disabled" href="contact.html">Contact</a>
+          <a class="nav-link" href="gallery.html">Photo Gallery</a>
+          <a class="nav-link" href="projects.html">Projects</a>
+        </nav>
+      </div>
+      <div id="content">
+        <div id="contact-information">
+          <h1>Contact Information</h1>
+          <div id="icons-container">
+            <i
+                class="fab fa-github fa-3x icon contact-icon"
+                onclick="openNewTab('https://github.com/lyskevin')">
```

Comment:
I think it is possible to wrap the <i> with <a> to make the icon clickable, have you tried that?

Reply:
Thanks for the feedback! I have tried wrapping it with an <a> tag along with a href but that changes the icons to be blue and underlined (like a normal link), and I would have to do custom CSS styling for the icons. I chose to go with the current implementation to avoid running into that issue, but I can do that if you think it is more suitable :)

Reply:
I see. yeah it seems to be the default behaviour, we can keep this for now